spoon-runner/src/main/java/com/squareup/spoon/SpoonRunner.java

```diff
@@ -288,7 +288,7 @@ private SpoonDeviceRunner getTestRunner(String serial, SpoonInstrumentationInfo
     private String methodName;
     private boolean noAnimations;
     private IRemoteAndroidTestRunner.TestSize testSize;
-    private int adbTimeout;
+    private int adbTimeout = DEFAULT_ADB_TIMEOUT;
```

Comment:
`DEFAULT_ADB_TIMEOUT ` is set in seconds. And this value is passed to `SpoonUtils.initAdb()` where timeout is expected to be in ms.